config/app.default.php

```diff
@@ -84,6 +85,7 @@
         'default' => [
             'className' => 'File',
             'path' => CACHE,
+            'url' => env('CACHE_DEFAULT_URL', ''),
```

Comment:
That's pretty fair. I named them this way because the naming is `_cache_model_`, which is pretty silly imo but *shrug* cake amirite?